package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ebay/skin",
   "description": "Skin - CSS modules that represent the brand eBay",
-  "version": "9.2.0",
+  "version": "9.3.0-1",
```

Comment:
When do we update this to `9.3.0`? When it's in master?

Reply:
Yes, I will checkout from master and then do the version changes and push directly to master.